build.gradle.kts

```diff
@@ -15,6 +15,7 @@ application {
 }
 
 repositories {
+    mavenLocal()
     mavenCentral()
     google()
```

Comment:
```suggestion
    mavenCentral()
    google()
```

Reply:
No good? How do you handle running locally against locally-built `revanced-library`?

Reply:
Locally you may add mavenLocal() however, only locally.

Reply:
Cool, dropped that commit.